testing-with-jest.test.js

```diff
@@ -9,19 +9,73 @@
 // --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
 
 
+describe("addThemUpNo", () => {
+    test( "add two numbers" , () => {
+      expect(addThemUp()).toEqual(16)
+        })
+})
+
+var numb1 = 10
+var numb2 = 5
+
+const addThemUpNo =  () => {
+    { return numb1 + numb2} 
+}
```

Comment:
This, unfortunately, is not correct. 

First, the function is named incorrectly. Second does not have any arguments, we would need to provide the function arguments like so:
`const addThemUp = (num1, num2) => { logichere }`

Third, you wrapped your return in curly braces, which I'm not sure why. 
Fourth, your return hard codes the variables you created above, which are unnecessary and also makes the function ONLY work with those two variables. Instead, your return should look something like:
`return num1 + num2` as those would be the arguments being passed into it.